src/GAP.jl

```diff
@@ -90,10 +90,19 @@ run_it = function(gapdir::String)
     initialize( [ ""
                        , "-l", sysinfo["GAP_LIB_DIR"]
```

Comment:
It should point to the standard GAP root as defined in `sysinfo.gap`, because it comes from the sysinfo. I am not sure it is really necessary. But it was also added by you in e56d23a769010d402f7703ab5ac1e871a3019636